src/main/java/com/flipkart/ranger/serviceprovider/ServiceProviderBuilder.java

```diff
@@ -132,14 +138,19 @@
             LOGGER.warn("Health update interval too low: {} ms. Has been upgraded to 1000ms ", healthUpdateIntervalMs);
             healthUpdateIntervalMs = 1000;
         }
+        if (staleUpdateThresholdMs < 1000) {
+            LOGGER.warn("Stale update too too low: {} ms. Has been upgraded to 60000ms ", staleUpdateThresholdMs);
+            staleUpdateThresholdMs = 60000;
```

Comment:
Also do the same if it is more than 60 secs .. somebody might want to get smart .. ;)

Reply:
Add the same check and fix to the healthUpdateIntervalMs as well. It is a bug right now.